naxsi_src/naxsi.h

```diff
@@ -357,6 +356,8 @@ typedef struct
   ngx_hash_t	*wlr_body_hash;
   /* hash table of whitelisted HEADERS rules */
   ngx_hash_t	*wlr_headers_hash;
+  /* hash table of pass HEADERS rules */
+  ngx_hash_t    *passr_headers_hash;
```

Comment:
Please align it with the same whitespaces used in the code.

Reply:
@wargio Thanks. Should be better now.